### PR TITLE
Clean up contract repos

### PR DESCRIPTION
**Pull request by 0xGabi**
This PR clean the current contract repos and build on #58 to use typescript + typechain for smart contract in both `govern-core` and `erc3k`. Existing tests were already migrated

### CODE REVIEW

File: packages/govern-core/test/pipelines/govern-queue.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+describe('Govern Queue', function () {
+  let owner: Signer, notOwner: Signer, ownerAddr, notOwnerAddr
+  let gq: GovernQueue
+
+  // TODO: add initial configuration
+  const INIT_CONFIG: any = {}
+
+  beforeEach(async () => {
+    ;[owner, notOwner] = await ethers.getSigners()
```
Comment (Evalir) [inline packages/govern-core/test/pipelines/govern-queue.test.ts]:
```suggestion
    [owner, notOwner] = await ethers.getSigners()
```

File: packages/govern-core/test/pipelines/govern-queue.test.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ownerAddr = await owner.getAddress()
+    notOwnerAddr = await notOwner.getAddress()
+
+    const GQ = (await ethers.getContractFactory(
```
Comment (Evalir) [inline packages/govern-core/test/pipelines/govern-queue.test.ts:34]: Do we need these parentheses around the `await ...`?

File: packages/govern-core/test/pipelines/govern-queue.test.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { ethers } from '@nomiclabs/buidler'
+import { Signer } from 'ethers'
+import { GovernQueue, GovernQueueFactory } from '../../typechain'
+
+describe('Govern Queue', function () {
+  let owner: Signer, notOwner: Signer, ownerAddr, notOwnerAddr
+  let gq: GovernQueue
+
+  // TODO: add initial configuration
+  const INIT_CONFIG: any = {}
```
Comment (Evalir) [inline packages/govern-core/test/pipelines/govern-queue.test.ts]: Hmmm should we do this now?

File: packages/govern-core/test/govern.test.ts [hunk 4]
```diff
@@ -20,7 +23,7 @@ describe('Govern', function () {
 
   beforeEach(async () => {
     const Govern = await ethers.getContractFactory('Govern')
-    govern = await Govern.deploy(owner)
+    govern = (await Govern.deploy(owner)) as Govern
```
Comment (Evalir) [inline packages/govern-core/test/govern.test.ts:26]: Ah, this seems to be a linting thing right (the parentheses)?

**Review by Evalir — APPROVED** [review]
This looks really nice! Apart from some questions, could we add some docs around how we're using typechain and how we're gonna use them for tests?

File: packages/govern-core/test/govern.test.ts (hunk 4 above)
Comment (kanaagawa) [reply packages/govern-core/test/govern.test.ts:26]: Yes it is

File: packages/govern-core/test/pipelines/govern-queue.test.ts (hunk 1 above)
Comment (kanaagawa) [reply packages/govern-core/test/pipelines/govern-queue.test.ts]: This was also a listing rule

**Pull request merged**